### PR TITLE
Save quant model

### PR DESCRIPTION
**Pull request by slf12**
Save quant model to avoid reshape after image input

### CODE REVIEW

**Review by ZeyuChen — APPROVED** [review]
LGTM

File: pdseg/models/model_builder.py [hunk 1]
```diff
@@ -175,12 +175,16 @@ def build_model(main_prog, start_prog, phase=ModelPhase.TRAIN):
...
             # 预测部署时只须对输入图像增加batch_size维度即可
             if ModelPhase.is_predict(phase):
-                origin_image = fluid.data(
-                    name='image',
-                    shape=[-1, -1, -1, cfg.DATASET.DATA_DIM],
-                    dtype='float32')
-                image, valid_shape, origin_shape = export_preprocess(
-                    origin_image)
+                if cfg.SLIM.PREPROCESS:
+                    image = fluid.layers.data(
```
Comment (ZeyuChen) [inline pdseg/models/model_builder.py]: Migrate `fluid.layers.data` to `fluid.data`

Comment (ZeyuChen) [inline pdseg/models/model_builder.py]: Migrate `fluid.layers.data` to `fluid.data`

File: pdseg/models/model_builder.py [hunk 2]
```diff
@@ -175,12 +175,16 @@ def build_model(main_prog, start_prog, phase=ModelPhase.TRAIN):
...
+                        name='image', shape=image_shape, dtype='float32')
+                else:
+                    origin_image = fluid.layers.data(
```
Comment (ZeyuChen) [inline pdseg/models/model_builder.py]: The same as the previous review.

**Review by ZeyuChen — CHANGES_REQUESTED** [review]
Change `fluid.layers.data` to `fluid.data` to upgrade our API to PaddlePaddle 1.7 standard.

File: pdseg/models/model_builder.py (hunk 1 above)
Comment (slf12) [reply pdseg/models/model_builder.py]: done

File: pdseg/models/model_builder.py (hunk 2 above)
Comment (slf12) [reply pdseg/models/model_builder.py]: done

**Pull request merged**